packages/legend-extension-dsl-serializer/src/components/studio/BindingElementEditor.tsx

```diff
@@ -0,0 +1,485 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+import type { SelectComponent } from '@finos/legend-art';
+import {
+  clsx,
+  CustomSelectorInput,
+  ResizablePanel,
+  ResizablePanelGroup,
+  ResizablePanelSplitter,
+  ResizablePanelSplitterLine,
+  TimesIcon,
+} from '@finos/legend-art';
+import { guaranteeNonNullable, prettyCONSTName } from '@finos/legend-shared';
+import type {
+  ElementDragSource,
+  UMLEditorElementDropTarget,
+} from '@finos/legend-studio';
+import {
+  CORE_DND_TYPE,
+  useEditorStore,
+  getElementIcon,
+} from '@finos/legend-studio';
+import {
+  BINDING_TAB_TYPE,
+  BindingEditorState,
+} from '../../stores/studio/BindingEditorState';
+import { FaLock } from 'react-icons/fa';
+import { SchemaSet } from '../../models/metamodels/pure/model/packageableElements/schemaSet/SchemaSet';
+import type {
+  OptionalPackageableElementReference,
+  PackageableElement,
+  PackageableElementReference,
+} from '@finos/legend-graph';
+import { PackageableElementExplicitReference } from '@finos/legend-graph';
+import { useCallback, useRef } from 'react';
+import { BINDING_CONTENT_TYPE } from '../../models/metamodels/pure/model/packageableElements/store/Binding';
+import { useDrop } from 'react-dnd';
+import type { ModelUnit } from '../../models/metamodels/pure/model/packageableElements/store/ModelUnit';
+
+enum BINDING_MODEL_UNIT {
+  ELEMENT_INCLUDE = 'ELEMENT_INCLUDE',
+  ELEMENT_EXCLUDE = 'ELEMENT_EXCLUDE',
+}
+
+const BindingScopeEntryEditor = observer(
+  (props: {
+    elementRef: PackageableElementReference<PackageableElement>;
+    removeElement: (
+      val: PackageableElementReference<PackageableElement>,
+      label: BINDING_MODEL_UNIT,
```

Comment:
no, let's not pass a `label`, this is no difference than what we had before, instead, pass different functions in for exclude and include case, please also update the one you have below accordingly